_sass/_donation-banner.scss

```diff
@@ -0,0 +1,174 @@
+$orange: #ef9e5b;
+$dark-orange: #de8236;
+$green: #4cad2c;
+$dark-green: #318f20;
+
+@mixin gradient($color1, $color2) {
+    background-image: -o-linear-gradient(top, $color1 14%, $color2 70%);
+    background-image: -moz-linear-gradient(top, $color1 14%, $color2 70%);
+    background-image: -webkit-linear-gradient(top, $color1 14%, $color2 70%);
+    background-image: -ms-linear-gradient(top, $color1 14%, $color2 70%);
+    background-image: linear-gradient(top, $color1 14%, $color2 70%);
+}
+
+.donation-btn {
+    background-color: #4CAF50;
+    @include gradient($green, $dark-green);
+    border: none;
+    border-radius: 10px;
+    padding: 10px 20px;
+    color: white;
+    font-weight: bold;
+}
+
+.donation-container {
+    text-align: center;
+
+    .donation-text {
+        background-color: #ef9e5b;
+        color: #ffffff;
+        font-weight: bold;
+        transition: margin-top .2s ease-in;
+        height: 90px;
+        margin-top: -110px;
+        padding: 10px;
+
+        &.expanded {
+            margin-top: 0;
+            border-bottom: 4px solid #ef9e5b;
+            box-shadow: 0 5px 10px rgba(0, 0, 0, 0.22);
```

Comment:
I'd offset the shadow 2px lower, in the live preview you can (barely) see the shadow of the pulldown over the expanded banner.